React/wbooks/src/app/screens/Home/components/Combobox/index.js

```diff
@@ -4,15 +4,12 @@ import PropTypes from 'prop-types';
 import styles from './styles.scss'
 
 class Combobox extends Component {
-
-  handleSelection = event => {
-    this.props.onSelection(event.target.value);
-  }
+  onSelectionChange = (event) => this.props.onSelection(event.target.value);
 
   render() {
     return (
       <div className={styles.comboboxContainer}>
-        <select className={styles.comboboxSelector} onChange={this.handleSelection}>
+        <select className={styles.comboboxSelector} onChange={(event) => this.onSelectionChange(event)}>
```

Comment:
you didn't correct this. Dont dont dont dont use arrow functions ir render methods